spec/models/profile_spec.rb

```diff
@@ -1003,12 +1003,37 @@
 
   describe '#deactivate_due_to_in_person_verification_cancelled' do
     let(:profile) { create(:profile, :in_person_verification_pending) }
-    it 'updates the profile' do
-      profile.deactivate_due_to_in_person_verification_cancelled
+    context 'updates the profile' do
```

Comment:
I think it might be nice to replace the `updates the profile` context to be two contexts one for `when the profile has a deactivation reason` and the other for `when the profile does not have a deactivation reason`. I think this will improve test readability and structure.

```ruby
describe '#deactivate_due_to_in_person_verification_cancelled' do
  let(:profile) { create(:profile, :in_person_verification_pending) }
  
  context 'when the profile has a deactivation reason' do
    # Note you might not need both of these because we only need to 
    # ensure the tests work if one exists
    context 'when the deactivation reason is "encryption_error"' do
      it 'updates the profile without overwriting the deactivation reason' do
      end
    end
    
    context 'when the deactivation reason is "password_reset"' do
      it 'updates the profile without overwriting the deactivation reason' do
      end
    end
  end
  
  context 'when the profile does not have a deactivation reason' do
    it 'updates the profile with deactivation reason set to "verification_cancelled"' do
    end
  end
end
```

Reply:
🤔  Do you think these changes make this method feel more focused on this attribute than on all of the updates? That is why I kept "updates the profile".  Let me see if I can think of another way to write these out.

Reply:
Those updates look good to me 👍🏻